src/modules/Experiences/AddOrEditExperienceForm/AddOrEditExperienceForm.js

```diff
@@ -183,7 +183,7 @@ const AddOrEditExperienceForm = ({
             value={degree}
             onChange={handleChange}
             name="degree"
-            label="Type of Education"
+            label="Type of Education *"
```

Comment:
In the medium-long run, it would be great to have these "*" be autogenerated by the validation code, and somehow noted here. That would prevent usability errors where something is mistakenly marked required when it's really not, or vice versa